### ISSUE TITLE
Button and Notification Banner: Update colours used or font size

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#2363]
# Info

This was raised by the UAIM Accessibility Report. The issue is a level 2 issue. "White text on blue is sometimes hard to read, font should be a little bigger". The recommendation is "White font for the foreground and blue background for a button can pass accessibility but when the font size is too small it does make the white font challenging to read. Increase the size of the font".

## Acceptance Criteria:

1. Button
    * Modify font size **OR**
    * Update colours for Primary, Start, and Submit buttons
3. Notification Banner
    * Modify font size **OR**
    * Update colours for Information and Event Notifications

**Comment by twjeffery** [comment]
The font size in our current component is smaller than designed. (small BUG) This might help, and we can retest this again after.

This has been fixed in this PR: https://github.com/GovAlta/ui-components/pull/2296

**Current**

<img width="190" alt="Image" src="https://github.com/user-attachments/assets/4ec88738-a9f9-422d-9791-cc92c4b4cb93" />

**Updated**
<img width="190" alt="Image" src="https://github.com/user-attachments/assets/3a1acfd1-fa6a-4fab-83a7-a4bc30594c27" />

**Comment by ArakTaiRoth** [comment]
This has been resolved with our redesign
